### PR TITLE
Regal Rat - Maint Creatures DLC Update

### PR DESCRIPTION
**Pull request by Comxy**
## About The Pull Request
The regal rat can now convert frogs and cockroaches to their army. They also will not attack each other with the new maintenance creature faction. BirdBoat has also been added to this faction since he also naturally appears in the maintenance.
## Why It's Good For The Game
Make sure to avoid stepping on the cockroaches. They do have one extra health though.
promo
## Changelog
:cl:
add: Frogs and cockroaches can now be converted to the regal rats army.
fix: Converted rat do not attack mouses anymore.
imageadd: New sewer cockroach and trash frog sprites
/:cl:

### CODE REVIEW

**Comment by Comxy** [comment]
I don't know why the test failed but it should work fine. Test merge and merge ready

**Comment by anclandor** [comment]
Does this move spit drinking from rat faction to maint creature faction?

**Comment by Comxy** [comment]
> Does this move spit drinking from rat faction to maint creature faction?

I don't think so, I haven't tempered anything with the rat faction

**Comment by ArcaneMusic** [comment]
> I don't know why the test failed but it should work fine. Test merge and merge ready

We're moving away from adding new simple_mobs and moving instead towards moving all existing simple_mobs to basic_mobs. That's what's causing all your tests to fail (Trash frog is still simple)

File: code/modules/mob/living/basic/vermin/cockroach.dm [hunk 1]
```diff
@@ -144,3 +144,58 @@
...
+	obj_damage = 10
+	gold_core_spawnable = NO_SPAWN
+	ai_controller = /datum/ai_controller/basic_controller/cockroach/glockroach
+	cockroach_cell_line = CELL_LINE_TABLE_GLOCKROACH
+
+/mob/living/basic/cockroach/sewer/glockroach/Initialize(mapload)
+	. = ..()
+	AddElement(/datum/element/ranged_attacks, /obj/item/ammo_casing/glockroach)
+
+/mob/living/basic/cockroach/sewer/hauberoach
```
Comment (LemonInTheDark) [inline code/modules/mob/living/basic/vermin/cockroach.dm:188]: please do not make sewer subtypes of every maint mob, this is a really bad pattern and you need to find a better way of doing this

**Comment by Comxy** [comment]
Regal rats will not attack each other but will attack each others armys.

**Review by Twaticus — APPROVED** [review]

**Review by Mothblocks — CHANGES_REQUESTED** [review]
No more simple mob subtypes, please remove the frogs until they are made into basic mobs

**Comment by Mothblocks** [comment]
@LemonInTheDark Can you look at this and see if you are fine with it now

**Comment by Mothblocks** [comment]
It's me Lemon on Mothblocks computer I think this is cool

**Comment by LemonInTheDark** [comment]
> It's me Lemon on Mothblocks computer I think this is cool

So true past me. he even removed the weird cockroach pathing so surely it's fine

**Pull request merged**